.github/workflows/release.yml

```diff
@@ -72,12 +72,12 @@ jobs:
 
       - name: Current Date
         id: date
-        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+        run: echo "date=$(date +'%Y-%m-%d')" >> GITHUB_OUTPUT
```

Comment:
Does this work without a dollar prefix? e.g. $GITHUB_OUTPUT

Reply:
As a whole it was working, if you look at the CI for my deluge PRs, but it shouldn't be. Well caught.

Reply:
Fixed in commit below